src/scss/partials/variables/bootstrap/_variable-overrides.scss

```diff
@@ -1,7 +1,6 @@
 // IT IS FORBIDDEN TO USE REFERENCES TO BS5 VARIABLES BECAUSE THEY ARE NOT YET DECLARED.
```

Comment:
The way I always do it is copy all bootstrap _variables.scss into theme and use this **instead** original variables, not before them. This way you can use references to BS5 variables already declared in this file without problems. Plus it gives better overview of what can be changed within BS theme variables and helps to keep the project clean, without duplicated code and useless variables like it was with classic.

Reply:
This is something we should discuss in a proper issue, and it will probably change a lot of files in our variable folder